unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -963,7 +963,7 @@ loop = do
                   types' = Set.map go types
                     where
                       go r = (r, NamesWithHistory.typeName hqLength r printNames)
-              respond $ ListNames hqLength (toList types') (toList terms')
+              respond $ ListNames global hqLength (toList types') (toList terms')
```

Comment:
`ListNames` picks up an `IsGlobal` flag as well, so it can print a helpful tip about using `names.global` or not, if the user has already just used `names.global`.